uwblueprint/community-fridge-kw
# feat: Scheduling Donation Information page

## Notion ticket link

[Scheduling Flow - Donation Information](https://www.notion.so/uwblueprintexecs/Team-Hub-5b0b225bfc09415ebcfad3030ce5fd30?p=bf3271a1deb14e22b6f0d2651cce169c)



## Implementation description
* implemented a custom radio button component for the images (potential refactor in future to add ternary operator between this and @AmandaMisjuwar's radio button ticket, potential refactor to handle updating a user's selection of checkboxes more elegantly... :) )
* implemented checkboxes
* selections passed to formValues so they can be retained across pages
* fixed spacing around the progress bar component 
* update global colours



## Steps to test
1. Navigate to the scheduling donation information page 
2. verify that selecting and unselecting items is functional 
3. navigate between pages to ensure selections remain 


## What should reviewers focus on?
* functionality 
* styling


## Checklist
